src/sql/parts/notebook/cellViews/codeActions.ts

```diff
@@ -1,21 +1,13 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
- *  Licensed under the Source EULA. See License.txt in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
 import { Action } from 'vs/base/common/actions';
 import { TPromise } from 'vs/base/common/winjs.base';
-import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
-import { IContextViewProvider } from 'vs/base/browser/ui/contextview/contextview';
-import { ISelectBoxOptions } from 'vs/base/browser/ui/selectBox/selectBox';
-import { TaskHistoryViewlet } from 'sql/parts/taskHistory/viewlet/taskHistoryViewlet';
 
 export class RunCellAction extends Action {
 	public static ID = 'jobaction.notebookRunCell';
 	public static LABEL = 'Run cell';
 
 	constructor(
 	) {
-		super(RunCellAction.ID, RunCellAction.LABEL, 'newStepIcon');
+		super(RunCellAction.ID, '', 'toolbarIconRun');
```

Comment:
We need these icons to have a title on hover. Can you check that this will work? I think we need to fix the CSS so in vertical mode we show the label as the on-hover title.

Reply:
There's a simple tooltip property that we can leverage here.